src/main.rs

```diff
@@ -250,8 +250,8 @@ pub fn setup_csrs() {
     hie.modify(Interrupt::VirtualSupervisorExternal.to_hie_field().unwrap());
     CSR.hie.set(hie.get());
 
-    // Make counters available to guests.
-    CSR.hcounteren.set(0xffff_ffff_ffff_ffff);
+    // Make timer available to guests.
```

Comment:
Probably should file a bug and leave a TODO here about `time` virtualization and `htimedelta` handling.